### PR TITLE
Use logging lib for logs configuring

### PR DESCRIPTION
**Pull request by vgarvardt**
* Use `github.com/hellofresh/logging-go` for logging configuring
* Made logging `logrus` 1.0 compatible

### CODE REVIEW

File: cmd/janus/init.go [hunk 1]
```diff
@@ -31,16 +27,12 @@ func init() {
 
 // initializes the basic configuration for the log wrapper
 func init() {
-	level, err := log.ParseLevel(strings.ToLower(globalConfig.LogLevel))
-	if err != nil {
-		log.WithError(err).Error("Error getting log level")
+	err := globalConfig.Log.Apply()
+	if nil != err {
+		if nil != err {
```
Comment (rafaeljesus) [inline cmd/janus/init.go]: Double checking `err` here

**Review by rafaeljesus — APPROVED** [review]

**Comment by rafaeljesus** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/hellofresh/api-gateway/pull-request/154/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/api-gateway) [Approved with PullApprove](https://pullapprove.com/hellofresh/api-gateway/pull-request/154/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/api-gateway)

**Comment by vgarvardt** [comment]
Rebased, @rafaeljesus  and @italolelis pls take a look again

**Comment by italolelis** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/hellofresh/api-gateway/pull-request/154/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/api-gateway) [Approved with PullApprove](https://pullapprove.com/hellofresh/api-gateway/pull-request/154/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/api-gateway) [Approved with PullApprove](https://pullapprove.com/hellofresh/api-gateway/pull-request/154/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/api-gateway)

**Pull request merged**